.changeset/tasty-weeks-drum.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/segmented-control': minor
+---
+
+[LG-4045](https://jira.mongodb.org/browse/LG-4045): Adds `'small'` size styling. Previously, `'small'` size would fall back on the `'xsmall'` sizeg styling
```

Comment:
I think this is breaking since things that were previously small will be a different size now